packages/vkui/src/components/FormField/FormField.module.css

```
@@ -196,11 +201,15 @@
   border-bottom-right-radius: 0;
 }
 
-:global(.vkuiInternalFormLayoutGroup--segmented)
-  > .FormItem:not(:first-of-type):last-of-type
+:global(
+    .vkuiInternalFormLayoutGroup--segmented > .vkuiInternalFormItem:not(:first-of-type):last-of-type
+  )
```

```suggestion
:global(.vkuiInternalFormLayoutGroup--segmented) > :global(.vkuiInternalFormItem):not(:first-of-type):last-of-type
  )
```
так читабельней кмк

---

коммент и для других подобных стилей в этом файле

---

Сделал